src/librustc_resolve/lib.rs

```diff
@@ -484,8 +484,8 @@ type BindingMap = FxHashMap<Ident, BindingInfo>;
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
 enum PatternSource {
     Match,
-    IfLet,
-    WhileLet,
+    // FIXME(54883): Consider fusing with `Let` below once let-statements support or-patterns.
```

Comment:
<3

Reply:
<3

Reply:
Also cc @dlrobertson; I've left this FIXME for your benefit... =P

Reply:
:+1: Thanks!